public/images/themes/light/logo.svg

```diff
@@ -1 +1,103 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1719 522"><defs><style>.a{fill:#2970ff;}.b{fill:#2dd8a3;}.c{fill:#091440;}</style></defs><title>transparent background - primary - positive - full color</title><path class="a" d="M291.62,200.75,232.87,142h63.45l94,94H326.87a47,47,0,0,0,0,94h23.5l47,47h-70.5a94,94,0,0,1-94-94C232.87,244.88,251.67,214.15,291.62,200.75Z"/><path class="b" d="M456.12,318.25,514.87,377H451.42l-94-94h63.45a47,47,0,0,0,0-94h-23.5l-47-47h70.5a94,94,0,0,1,94,94C514.87,274.12,496.07,304.85,456.12,318.25Z"/><path class="c" d="M914,305.75v-.4C914,263.68,946,231,989.32,231c29.21,0,48,12.26,60.66,29.82l-29.82,23.08c-8.17-10.22-17.56-16.75-31.24-16.75-20,0-34.12,17-34.12,37.79v.41c0,21.44,14.1,38.19,34.12,38.19,14.9,0,23.69-7,32.27-17.36L1051,347.42c-13.48,18.59-31.66,32.27-63.32,32.27C946.84,379.69,914,348.45,914,305.75Z"/><path class="c" d="M1065.13,232.84h68.09c22,0,37.23,5.73,46.9,15.6,8.44,8.23,12.75,19.34,12.75,33.54v.41c0,22-11.72,36.61-29.62,44.23l34.36,50.22h-46.08l-29-43.64H1105v43.64h-39.91Zm66.24,69.09c13.58,0,21.39-6.59,21.39-17.08v-.41c0-11.31-8.23-17.07-21.59-17.07H1105v34.56Z"/><path class="c" d="M1211.6,232.84h115.81v33.94H1251.1v21.8h69.11v31.48H1251.1v22.83h77.34v33.95H1211.6Z"/><path class="c" d="M1349.34,232.84h55.54c51.42,0,81.25,29.62,81.25,71.17v.42c0,41.55-30.24,72.41-82.08,72.41h-54.71Zm56.16,108.61c23.86,0,39.7-13.16,39.7-36.41v-.41c0-23-15.84-36.41-39.7-36.41h-16.25v73.23Z"/><path class="c" d="M832.56,379.53c-42.27,0-72.94-31.14-72.94-74,0-42.37,29.81-74.33,69.36-74.33,21.18,0,38.77,7.89,50.86,22.82,11,13.61,17.11,32.79,17.11,54,0,2.45-.18,4.72-.4,7.31H789.09l.34,2.08c3.86,23.72,20.6,38.46,43.68,38.46,15.19,0,27.08-5,39.49-16.73l17,15.11C874.85,371.24,856.17,379.53,832.56,379.53ZM828.7,254.87c-20.78,0-36.31,15.66-39.56,39.89l-.27,2h78.77l-.2-2C864.91,270.18,850.07,254.87,828.7,254.87Z"/><path class="c" d="M659.23,379.53c-32.81,0-66-25.53-66-74.33s33.21-74.34,66-74.34c19.5,0,35.34,8.2,48.41,25l3.22,4.15V166.91h29.9V376.84h-29.9V348.28l-3.26,4.6C694.88,370.81,679.06,379.53,659.23,379.53Zm8-123.19c-25.83,0-43.87,20.09-43.87,48.86,0,28.31,18.45,48.86,43.87,48.86,25.17,0,44.15-21,44.15-48.86S692.43,256.34,667.26,256.34Z"/></svg>
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I've been cleaning up the svg files with [`svgcleaner`](https://github.com/RazrFalcon/svgcleaner) to compress the files:

```sh
$  ll logo.svg
-rw-r--r-- 1 jon users 4989 Nov 12 09:05 logo.svg
$  svgcleaner --indent=tabs logo.svg logo.svg
Your image is 42.73% smaller now.
$  ll logo.svg
-rw-r--r-- 1 jon users 2857 Nov 12 09:03 logo.svg
``

I suggest we keep doing that unless it's preferable to have the verbose svg files.